lib/mongo_mapper/eager_includer.rb

```diff
@@ -50,18 +50,14 @@ def eager_include
 
   attr_reader :association_name
 
-  def instance_variable_name
-    "@eager_loaded_#{association_name}"
+  def get_association_proxy(record)
+    record.send(:get_proxy, @association)
```

Comment:
Or better yet - record.associations[@association.name] I *think* will also get the proxy.